hooks/stateupdates/patches.go

```diff
@@ -7,8 +7,8 @@ import (
 var stateupdatesPatchsets = []xplugeth.Patchset {
 	xplugeth.Patchset{
 		Remote: "github.com/openrelayxyz/xplugeth-patches",
-		Ref: "hooks_foundation_stateupdates_v1.15.6_1",
-		// https://github.com/openrelayxyz/xplugeth-patches/commit/36beb3fd98c77aded8964cb5ff5965b870918d2f
+		Ref: "hooks_foundation_stateupdates_v1.16.4_0",
```

Comment:
Let's add this as an additional patch, instead of a replacement. I believe that will keep the bor build working a while longer.

Reply:
Good call. It will also keep our current mainnet build viable on the default branch.